module/src/hooks/form/form.utils.ts

```diff
@@ -75,7 +78,7 @@ export function valueByKeyChain<TData, TValue>(state: TData, keyChain: KeyChain)
  * @returns {boolean} `true` if the item passed conforms to the binding props interface `IBindingProps` else `false`. Also casts if `true`.
  */
 export function isBindingProps<TValue>(item?: any): item is IBindingProps<TValue> {
-  return !!item?.setValue && !!item?.dispatch && !!item?.keyChain && !!item?.myValidationErrors;
+  return !!item && 'setValue' in item && 'dispatch' in item && 'keyChain' in item && 'myValidationErrors' in item;
```

Comment:
Probably revert for consistency. Also, using `in` like this removes all type safety on the properties being accessed, so if `setValue` ever got renamed for example, the compiler would just let this through and a bug would be introduced with no warning.

Reply:
Fixed as discussed.